### ISSUE TITLE
[BUG] getting-started can not run well

### ISSUE THREAD

**Issue by wj-Mcat** [thu-coai/ConvLab-2#72]
**Describe the bug**
When I follow the `Installation ` step and run `tutorials/Getting_Started.ipynb`, It gets into trouble.

**To Reproduce**
1. `conda create -n convlab-2`
2. `pip install -e .`
3. `jupyter notebook`
4. run the `tutorials/Getting_Started.ipynb`

**Expected behavior**
It will run well for me.

**Additional context**
```text
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-765703729d4c> in <module>
      1 # common import: convlab2.$module.$model.$dataset
----> 2 from convlab2.nlu.jointBERT.multiwoz import BERTNLU
      3 from convlab2.nlu.milu.multiwoz import MILU
      4 from convlab2.dst.rule.multiwoz import RuleDST
      5 from convlab2.policy.rule.multiwoz import RulePolicy

~/vscode/dialogue/ConvLab-2/convlab2/nlu/jointBERT/multiwoz/__init__.py in <module>
----> 1 from convlab2.nlu.jointBERT.multiwoz.nlu import BERTNLU

~/vscode/dialogue/ConvLab-2/convlab2/nlu/jointBERT/multiwoz/nlu.py in <module>
      7 from convlab2.util.file_util import cached_path
      8 from convlab2.nlu.nlu import NLU
----> 9 from convlab2.nlu.jointBERT.dataloader import Dataloader
     10 from convlab2.nlu.jointBERT.jointBERT import JointBERT
     11 from convlab2.nlu.jointBERT.multiwoz.postprocess import recover_intent

~/vscode/dialogue/ConvLab-2/convlab2/nlu/jointBERT/dataloader.py in <module>
      2 import torch
      3 import random
----> 4 from transformers import BertTokenizer
      5 import math
      6 from collections import Counter

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/__init__.py in <module>
     46 from .configuration_xlm_roberta import XLM_ROBERTA_PRETRAINED_CONFIG_ARCHIVE_MAP, XLMRobertaConfig
     47 from .configuration_xlnet import XLNET_PRETRAINED_CONFIG_ARCHIVE_MAP, XLNetConfig
---> 48 from .data import (
     49     DataProcessor,
     50     InputExample,

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/data/__init__.py in <module>
      4 
      5 from .metrics import is_sklearn_available
----> 6 from .processors import (
      7     DataProcessor,
      8     InputExample,

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/data/processors/__init__.py in <module>
      3 # module, but to preserve other warnings. So, don't check this module at all.
      4 
----> 5 from .glue import glue_convert_examples_to_features, glue_output_modes, glue_processors, glue_tasks_num_labels
      6 from .squad import SquadExample, SquadFeatures, SquadV1Processor, SquadV2Processor, squad_convert_examples_to_features
      7 from .utils import DataProcessor, InputExample, InputFeatures, SingleSentenceClassificationProcessor

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/data/processors/glue.py in <module>
     23 
     24 from ...file_utils import is_tf_available
---> 25 from ...tokenization_utils import PreTrainedTokenizer
     26 from .utils import DataProcessor, InputExample, InputFeatures
     27 

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/tokenization_utils.py in <module>
     24 
     25 from .file_utils import add_end_docstrings
---> 26 from .tokenization_utils_base import (
     27     ENCODE_KWARGS_DOCSTRING,
     28     ENCODE_PLUS_ADDITIONAL_KWARGS_DOCSTRING,

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/transformers/tokenization_utils_base.py in <module>
     29 
     30 import numpy as np
---> 31 from tokenizers import AddedToken
     32 from tokenizers import Encoding as EncodingFast
     33 

~/miniconda3/envs/convlab-2/lib/python3.7/site-packages/tokenizers/__init__.py in <module>
     15 EncodeInput = Union[TextEncodeInput, PreTokenizedEncodeInput]
     16 
---> 17 from .tokenizers import Tokenizer, Encoding, AddedToken
     18 from .tokenizers import decoders
     19 from .tokenizers import models

ImportError: dlopen(/Users/wujingwujing/miniconda3/envs/convlab-2/lib/python3.7/site-packages/tokenizers/tokenizers.cpython-37m-darwin.so, 2): Symbol not found: ____chkstk_darwin
  Referenced from: /Users/wujingwujing/miniconda3/envs/convlab-2/lib/python3.7/site-packages/tokenizers/tokenizers.cpython-37m-darwin.so (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/wujingwujing/miniconda3/envs/convlab-2/lib/python3.7/site-packages/tokenizers/tokenizers.cpython-37m-darwin.so
```

**Comment by zqwerty** [comment]
fixed #74

**Comment by wj-Mcat** [comment]
Glad to see that.
